proto/observer/ballot.proto

```diff
@@ -30,5 +30,11 @@ message Ballot {
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
     (gogoproto.nullable) = false
   ];
-  BallotStatus ballot_status = 7;
+  BallotStatus ballot_status =7;
+  int64 Ballot_creation_height=8;
```

Comment:
space, also naming convention should be at least not mixed upper case and underscore

Reply:
Added a proto formatter to make file and fixed the names